openslides_backend/action/generics.py

```diff
@@ -307,7 +307,7 @@ def delete_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
         """
 
         data = []
-        for instance in payload:
+        for instance in self.update_instances(payload):
```

Comment:
Please rebase to get the newest changes here

Reply:
Rebased.